validator/src/main.rs

```diff
@@ -588,6 +596,12 @@ pub fn main() {
     let mut validator_config = ValidatorConfig::default();
     validator_config.dev_sigverify_disabled = matches.is_present("dev_no_sigverify");
     validator_config.dev_halt_at_slot = value_t!(matches, "dev_halt_at_slot", Slot).ok();
+    if matches.is_present("hard_forks") {
+        for slot in values_t_or_exit!(matches, "hard_forks", Slot).into_iter() {
+            let entry = validator_config.hard_forks.entry(slot).or_insert(0);
+            *entry += 1;
```

Comment:
I want to confirm... Repeating the same `--ford-fork 232` or not should result in different bank hash?

Reply:
It should be a different bank hash.  If we try to resume at slot 1234 and fail the first time I want to be able to attempt to resume from slot 1234 a second time.   This would be done by just adding an additional `--hard-fork` arg for the same slot number

Reply:
I see. IMO, repeating `--hard-fork 1234` just looks a type, so might not be super user-friendly. Yet, this is cosmetic thing, which can be improved anytime. :) So, I'm fine as-is.

Reply:
Deal :)